core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -26,8 +26,10 @@ oppia.directive('simpleEditorSidebar', [
       controller: [
         '$scope', 'EditorModeService', 'SimpleEditorManagerService',
         'ScrollSyncService', 'QuestionIdService',
+        'SimpleEditorSidebarModeService',
         function($scope, EditorModeService, SimpleEditorManagerService,
-                 ScrollSyncService, QuestionIdService) {
+                 ScrollSyncService, QuestionIdService,
+                  SimpleEditorSidebarModeService) {
```

Comment:
I have de indented it according to start of 'function'. please have a look and let me know if that's right.